boards/fanke/fk7b0m1_vbt6/fk7b0m1_vbt6.dts

```diff
@@ -75,6 +76,61 @@
 	d3ppre = <2>;
 };
 
+&octospi1 {
+	pinctrl-0 = <&octospim_p1_clk_pb2 &octospim_p1_ncs_pb6
+		     &octospim_p1_io0_pd11 &octospim_p1_io1_pd12
+		     &octospim_p1_io2_pe2 &octospim_p1_io3_pd13>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	/* Winbond external flash */
+	w25q64jvssiq_qspi: qspi-nor-flash@0 {
+		compatible = "st,stm32-ospi-nor";
+		reg = <0>;
+		ospi-max-frequency = <DT_FREQ_M(133)>;
+		size = <DT_SIZE_M(64)>; /* 64 Mbits */
+		spi-bus-width = <OSPI_QUAD_MODE>;
+		data-rate = <OSPI_STR_TRANSFER>;
+		writeoc = "PP_1_1_4";
+		status = "okay";
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			slot0_partition: partition@0 {
+				reg = <0x00000000 DT_SIZE_M(64)>;
+			};
+		};
+	};
+};
+
+&spi6 {
+	pinctrl-0 = <&spi6_sck_pb3 &spi6_miso_pb4 &spi6_mosi_pb5>;
+	cs-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
+	pinctrl-names = "default";
+	status = "okay";
+	w25q64jvssiq_spi: spi-nor-flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <40000000>;
+		size = <DT_SIZE_M(64)>; /* 64 Mbits */
```

Comment:
Ditto on `size` prop

Reply:
@erwango, If a remove `size` and change to `reg = <0 DT_SIZE_M(8)>;` I got the error message
`error: static assertion failed: "jedec,spi-nor size required for non-runtime SFDP page layout"`

I'm up-to-date with the main branch (hash commit c6e482714f2387c).

Reply:
ok, my bad. This is indeed a different compatible.